charts/handbrake/2.0.0/questions.yaml

```diff
@@ -0,0 +1,797 @@
+                        default: 36052
+                        required: true
+        - variable: vnc
+          label: "VNC service"
+          description: "Service to connect to VNC if enabled"
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "Enable the service"
+                schema:
+                  type: boolean
+                  default: true
+                  hidden: true
+              - variable: type
+                label: "Service type"
+                description: "ClusterIP's are only internally available, nodePorts expose the container to the host node System"
+                schema:
+                  type: string
+                  default: "ClusterIP"
+                  enum:
+                    - value: "NodePort"
+                      description: "NodePort"
+                    - value: "ClusterIP"
+                      description: "ClusterIP"
+              - variable: port
+                label: "Port configuration"
+                schema:
+                  type: dict
+                  attrs:
+                    - variable: protocol
+                      label: "Port Type"
+                      schema:
+                        type: string
+                        default: "TCP"
+                        hidden: true
+                        enum:
+                          - value: TCP
+                            description: "TCP"
+                          - value: "UDP"
+                            description: "UDP"
+                    - variable: port
+                      label: "container port"
+                      schema:
+                        type: int
+                        default: 5900
+                        editable: false
+                        hidden: true
+                    - variable: targetport
+                      label: "Internal Service port"
+                      description: "When connecting internally to this App, you'll need this port"
+                      schema:
+                        type: int
+                        default: 5900
+                        editable: false
+                        hidden: true
+                    - variable: nodePort
+                      label: "(optional) host nodePort to expose to"
+                      description: "only get used when nodePort is selected"
+                      schema:
+                        type: int
+                        min: 9000
+                        max: 65535
+                        default: 36052
+                        required: true
+
+
+# Privileged
+  - variable: securityContext
+    group: "Storage and Devices"
+    label: "Security Context"
+    schema:
+      type: dict
+      attrs:
+        - variable: privileged
+          label: "Enable privileged to pass thru an Optical Drive device"
+          schema:
+            type: boolean
+            default: true
+            required: false
+
```

Comment:
```suggestion

```
Moving to the end of storage UI
(github wont add suggestion with many lines for some reason so i have to split it)